src/widgets/pixbuf_loader.rs

```diff
@@ -18,16 +18,12 @@ pub struct PixbufLoader {
 impl PixbufLoader {
     /// Creates a new pixbuf loader object.
     ///
-    /// # Failures
-    /// Returns `None` if the pixbuf loader cannot be created.
-    pub fn new() -> Option<PixbufLoader> {
+    /// # Panics
+    /// Fails if the pixbuf loader cannot be retrieved. 
+    pub fn new() -> PixbufLoader {
```

Comment:
This is tied up with returning `Result`s as well, so it seems you're saying don't change the APIs unless you're converting the whole crate?